main.go

```diff
@@ -0,0 +1,304 @@
+			Affiliation: "owner",
+		})
+		for _, r := range repos {
+			repo := Repo{r.GetOwner().GetLogin(), r.GetName()}
+			baseRepos = append(baseRepos, repo)
+		}
+	case "migrate":
+		// get spec file
+		resp, err := http.Get("https://bitrise-steplib-collection.s3.amazonaws.com/spec.json")
+		if err != nil {
+			panic(err)
+		}
+
+		// read spec file
+		spec, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		// unmarshal spec file
+		var data Spec
+		err = json.Unmarshal(spec, &data)
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		// process steps
+		for _, stp := range data.Steps {
+
+			// get latest version for step
+			url, ok := stp.Versions[stp.LatestVersionNumber]["source_code_url"].(string)
+			if !ok {
+				fmt.Println("could not convert json data")
+				os.Exit(1)
+			}
+
+			orgs := []string{
+				"bitrise-steplib",
+				"bitrise-io",
+				"bitrise-core",
+				"bitrise-community",
+				"bitrise-tools",
+				"bitrise-docker",
+				"bitrise-samples",
+			}
+
+			// filter to our repositories
+			for _, o := range orgs {
+				if strings.Contains(url, o) {
+					fragments := strings.Split(url, "/")
+					name := fragments[len(fragments)-1]
+					owner := fragments[len(fragments)-2]
+					repo := Repo{owner, name}
+					baseRepos = append(baseRepos, repo)
+				}
+			}
+
+		}
+		fmt.Printf("found %d repos, querying open issues", len(baseRepos))
+		fmt.Println()
+		fmt.Println()
+
+	}
+
+	// c, staleCount, activeCount, cPR := 0, 0, 0, 0
+	var err error
+	f, err = os.OpenFile(stateFile, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0644)
+	ferr, err = os.OpenFile("err.txt", os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0644)
+	defer f.Close()
+	defer ferr.Close()
+	if err != nil {
+		fmt.Printf("opening state file: %s", err)
+		os.Exit(1)
+	}
+
+	// get issues for repositories
+	opts := github.IssueListByRepoOptions{
+		State: "open",
+	}
+
+	switch mode {
+	case "test", "migrate":
+		for j, r := range baseRepos {
+			fmt.Println()
+			fmt.Println(strings.Repeat("=", 80))
+			fmt.Println(fmt.Sprintf("  processing repo (%d/%d): %s", len(baseRepos), j, r.Name))
+			fmt.Println(strings.Repeat("=", 80))
+			fmt.Println()
+
+			issues, _, _ := client.Issues.ListByRepo(ctx, r.Owner, r.Name, &opts)
+
+			process(issues)
+
+		}
+	case "continue":
+		// load state file
+		content, err := ioutil.ReadFile(stateFile)
+		if err != nil {
+			fmt.Println(fmt.Sprintf("could not read restore file: %s", err))
+			os.Exit(1)
+		}
+
+		// get (issue -> last state) map
+		lines := strings.Split(string(content), "\n")
+		issueStates := make(map[string]*RestoredIssue)
+		for _, l := range lines {
+			if len(l) == 0 {
+				continue
+			}
+
+			// parse line
+			fields := strings.Split(l, " ")
+
+			url := fields[0]
+			fragments := strings.Split(url, "/")
+
+			owner := fragments[3]
+			repo := fragments[4]
+			num64, err := strconv.ParseInt(fragments[6], 10, 8)
```

Comment:
if you need `int` . and you are sure that the values does not overflow the int bound (i guess it does not), use: `strconv. Atoi`